app/src/js/utils/format.js

```diff
@@ -71,6 +71,35 @@ export const tally = (numberstring) => {
   return bigTally(number);
 };
 
+export const fullStatTotal = (numberstring) => {
+  const number = +numberstring;
+  if (
+    (!numberstring && numberstring !== 0) ||
+    numberstring === nullValue ||
+    Number.isNaN(number)
+  ) {
+    return nullValue;
+  }
+
+  if (number >= 0) {
+    return `${numeral(number).format('0,0')}`;
```

Comment:
I think we can just return the output of the format function like this, since that returns a string
```suggestion
    return numeral(number).format('0,0');
```

Reply:
@dopeters Oh okay. Gotcha.

Regarding tests, I have never written a test so I'm not sure that works. 

Still trying to get my Bamboo access to see the lint errors.

Reply:
Ok no problem. Re: lint errors, you can do `npm run lint` locally and it should them to you

Reply:
Awesome, thanks @dopeters!